vncviewer/OptionsDialog.cxx

```diff
@@ -786,9 +798,50 @@ void OptionsDialog::createSecurityPage(int tx, int ty, int tw, int th)
   /* Back to normal */
   tx = orig_tx;
   ty += INNER_MARGIN;
+#endif // defined(HAVE_GNUTLS) || defined(HAVE_NETTLE)
+
+#ifndef WIN32
+  /* Connection */
+  ty += GROUP_LABEL_OFFSET;
+  //  height = GROUP_MARGIN * 2 + (INPUT_LABEL_OFFSET + INPUT_HEIGHT) * 1;
+  connectionGroup = new Fl_Group(tx, ty, width, 0, _("Connection"));
```

Comment:
"Connection" is a bit general, so I think we need something more specific here. "Gateway" is what's used in the documentation, so something including that word would probably be best for consistency.